packages/jest-config/package.json

```diff
@@ -36,7 +36,7 @@
     "chalk": "^4.0.0",
     "ci-info": "^3.2.0",
     "deepmerge": "^4.2.2",
-    "glob": "^7.1.3",
+    "glob": "^8.0.0",
```

Comment:
I wonder if this (and `jest-reporters` and `jest-runtime`) is unsafe if people today depend the globs working. Might need to save for Jest 29.

Updating in root is fine since that's only used in this repo internally